server/common/health.py

```diff
@@ -23,7 +23,11 @@ def health_check(config):
     """
     health = {"status": None, "version": "1", "releaseID": cellxgene_version}
 
-    checks = [_is_accessible(config.datapath), _is_accessible(config.dataroot)]
+    checks = [
+        lambda: config.single_dataset__datapath is not None or config.multi_dataset__dataroot is not None,
```

Comment:
I'm not sure I understand the lambda,   I think this produces a function , which always evaluates to True.
I think you just want just the expression: ```config.single_dataset__datapath is not None or config.multi_dataset__dataroot is not None```

Reply:
indeed.  thanks for catching!